Sources/TuistCore/Graph/Graph.swift

```diff
@@ -465,7 +465,7 @@ public class Graph: Graphing {
         return references
     }
 
-    public func findAll<T: GraphNode>(targetNode: TargetNode, test: (T) -> Bool = { _ in true }, skip: (T) -> Bool = { _ in false }) -> Set<T> {
+    public func findAll<T: GraphNode, S: GraphNode>(targetNode: TargetNode, test: (T) -> Bool = { _ in true }, skip: (S) -> Bool = { _ in false }) -> Set<T> {
```

Comment:
This change was needed to allow specifying `skip` rules for nodes of different types than the ones specified in `test`.

E.g. testing `PrecompiledNode` and skipping `TargetNodes`